### PR TITLE
Param as model

### PR DESCRIPTION
**Pull request by 0ssigeno**

### CODE REVIEW

File: api_app/core/serializers.py [hunk 1]
```diff
@@ -35,63 +21,155 @@ class _ConfigSerializer(rfs.Serializer):
...
+            else:
+                disabled = True
+            plugin_representation["disabled"] = disabled
+            plugin_representation["verification"] = {
+                "configured": configured,
+                "details": details,
+                "missing_secrets": parameter_required_not_configured,
+            }
+            result.append(plugin_representation)
+
```
Comment (mlodic) [inline api_app/core/serializers.py:162]: log of the number of generated plugin representations per user?

File: api_app/core/serializers.py [hunk 2]
```diff
@@ -35,63 +21,155 @@ class _ConfigSerializer(rfs.Serializer):
...
+            )
+        )
+        parsed = defaultdict(list)
+        # populate the result for every plugin (even the ones without parameters)
+        # parsed[plugin]= [parameter1]
+        for parameter in params:
+            parsed[getattr(parameter, self.child.Meta.model.snake_case_name)].append(
+                parameter
+            )
+
```
Comment (mlodic) [inline api_app/core/serializers.py:110]: log of user and number of params?

File: intel_owl/settings/__init__.py [hunk 3]
```diff
@@ -41,9 +41,9 @@
     # auth
     "rest_email_auth",
     "drf_recaptcha",
+    # performance debugging
```
Comment (mlodic) [inline intel_owl/settings/__init__.py:44]: leftover?

File: api_app/core/serializers.py (hunk 1 above)
Comment (0ssigeno) [reply api_app/core/serializers.py:162]: It is always the number of configurations present in the db. I just that some of them are disabled

**Pull request merged**